styles/AdvancedEditor.css

```diff
@@ -751,14 +774,23 @@ figure .selected-suggestion {
   box-shadow: inset 3px 0 0 var(--updated-border-color);
 }
 
+.tracking-visible .selected-suggestion[data-track-op='set_attrs'] .block,
+.tracking-visible .selected-suggestion [data-track-op='set_attrs'] .block,
+.tracking-visible .block:has(figure.selected-suggestion[data-track-op='set_attrs']),
+.tracking-visible .selected-suggestion[data-track-op='node_split'] .block,
+.tracking-visible figure.block:has(.equation.set_attrs.selected-suggestion) {
+  box-shadow: inset 6px 0 0 var(--updated-border-color), inset 9px 0 0 var(--updated-bg-color) !important;
+  animation: fadeOutBackground 3s forwards;
+  --fade-color: var(--updated-bg-color);
+}
+
 .selected-suggestion[data-track-op='set_attrs'] .block,
 .selected-suggestion [data-track-op='set_attrs'] .block,
 .block:has(figure.selected-suggestion[data-track-op='set_attrs']),
 .selected-suggestion[data-track-op='node_split'] .block,
 figure.block:has(.equation.set_attrs.selected-suggestion) {
-  box-shadow: inset 6px 0 0 var(--updated-border-color), inset 9px 0 0 var(--updated-bg-color) !important;
```

Comment:
There is no need for this change here. You can keep the old and just add the class `.tracking-visible` to the selectors .

Reply:
I revert it to master, do that update but that is still not working

Reply:
This is wired. I didn't add the previous lines you have 775-783 and kept the old selected block update styling with just adding `.tracking-visible` and it works fine.